### ISSUE TITLE
Error in ASR Training stage: Missing transformer_transducer

### ISSUE THREAD

**Issue by entn-at** [hirofumi0810/neural_sp#35]
Hi,
Thanks for creating such a great repo!
I tried running the SWBD example and ran into an issue at the ASR training stage. It seems a python module is missing (`transformer_transducer.py`)
```
============================================================================
                       ASR Training stage (stage:4)
============================================================================
Traceback (most recent call last):
  File "/home/user/neural_sp/neural_sp/bin/asr/train.py", line 41, in <module>
    from neural_sp.models.seq2seq.speech2text import Speech2Text
  File "/home/user/neural_sp/neural_sp/models/seq2seq/speech2text.py", line 26, in <module>
    from neural_sp.models.seq2seq.decoders.transformer_transducer import TrasformerTransducer
ModuleNotFoundError: No module named 'neural_sp.models.seq2seq.decoders.transformer_transducer'
```
Could you add the file, or should I just remove any reference to it in `speech2text.py`?
Thanks!

**Comment by hirofumi0810** [comment]
@entn-at Thank you for your report!
I fixed the bug in #40.

**Comment by entn-at** [comment]
Thanks!
